src/custom-gutter-component.coffee

```diff
@@ -0,0 +1,107 @@
+{setDimensionsAndBackground} = require './gutter-component-helpers'
+
+# This class represents a gutter other than the 'line-numbers' gutter.
+# The contents of this gutter may be specified by Decorations.
+
+module.exports =
+class CustomGutterComponent
```

Comment:
I like that `CustomGutterComponent` describes how it's used differently than `LineNumberGutterComponent`.